web/public/index.html

```diff
@@ -6,6 +6,8 @@
     <meta name="viewport" content="width=device-width,initial-scale=1.0">
     <link rel="icon" href="<%= BASE_URL %>favicon.ico">
     <title>dandi</title>
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:100,300,400,500,700,900">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@mdi/font@latest/css/materialdesignicons.min.css">
```

Comment:
This is not strictly necessary right now, since the MDI are [inlined by GWC](https://github.com/girder/girder_web_components/blob/2ee63d169cd21125654871afac85ee8c14714c0b/src/components/index.js#L1). However, we may drop GWC at some point. Alternatively, we may want to consider requesting an upstream change to GWC to make inclusion of the MDI a build-time requirement, and leave the implementation choice up to the downstream builder.